src/components/homepage/HeroSection.jsx

```diff
@@ -33,197 +31,169 @@ const EditorHeroSection = ({
   onFieldChange,
   createHandler,
   deleteHandler,
-  shouldDisplay,
   displayHandler,
   displayDropdownElems,
   displayHighlights,
   errors,
   handleHighlightDropdownToggle,
 }) => (
-  <div
-    className={`${elementStyles.card} ${
-      !shouldDisplay && !isEmpty(errors.sections[0].hero)
-        ? elementStyles.error
-        : ""
-    }`}
-  >
-    <div className={elementStyles.cardHeader}>
-      <h2>Hero section</h2>
-      <IconButton
-        variant="clear"
-        id={`section-${sectionIndex}`}
-        onClick={displayHandler}
-      >
-        <i
-          className={`bx ${
-            shouldDisplay ? "bx-chevron-down" : "bx-chevron-right"
-          }`}
-          id={`section-${sectionIndex}-icon`}
-        />
-      </IconButton>
-    </div>
-    {shouldDisplay ? (
-      <>
-        <FormContext isRequired hasError={!!errors.sections[0].hero.title}>
-          <FormTitle>Hero title</FormTitle>
-          <FormField
-            placeholder="Hero title"
-            id={`section-${sectionIndex}-hero-title`}
-            value={title}
-            onChange={onFieldChange}
+  <>
+    <FormContext isRequired hasError={!!errors.sections[0].hero.title}>
```

Comment:
wah, do you know why our shape is `errors.sections[0]` even though the shape only has one element `hero`? While this is out of scope of this pr, if during dev you find no such reason to use an array here, can shift to obj please! 

I am worried here that the original intention was to have an "array of errors" and we are only displaying the "first" error here

Reply:
I think it's because of the implicit assumption that the first section is the `hero` section - how the existing data structure works is that the homepage is represented by multiple sections, and each section is one of `hero`/`infopic`/`infobar`/`resources`, which will have that particular key only, so `sections[0]` always contains `hero`

Reply:
wait are you trying to say that `sections[1]` could be of type `infopic, infobar, resources`?

Reply:
^ yes, this is true. the existing code doesn't do a good job of fleshing this out but the invariant of `sections[0] === hero` is always maintained because we don't allow CRUD operations on it, only on subelements of it